test/all.test.ts

```diff
@@ -275,14 +387,58 @@ describe("remainder()", () => {
 });
 
 describe("divideAndRound()", () => {
-  it("works", () => {
+  it("rounds decimals", () => {
```

Comment:
:sweat_smile:

Reply:
haha I updated it because I added other tests in `divideAndRound()` so wanted to disambiguate between the cases

Reply:
haha yes much better than my `"works"` :smile: